client/src/language/diagnosticsSupport.ts

```diff
@@ -8,6 +8,7 @@ import * as vscode from 'vscode'
 import { getOriginalDocRange } from './utils'
 import { embeddedLanguageDocsManager } from './EmbeddedLanguageDocsManager'
 import { type EmbeddedLanguageType } from '../lib/src/types/embedded-languages'
+import { commonDirectoriesVariables } from '../lib/src/availableVariables'
```

Comment:
Now that this is handled in bash, you should try to provide it for bitbake variables assignments:
 - completion
 - value/definition is already provided by Ziwei after recipe scan
`FILES:${PN} = "${sysconfdir}/udhcpc.d"`